src/Mvc/Model/Migration.php

```diff
@@ -319,6 +319,11 @@ public static function generate(
 
             if ($field->isNotNull()) {
                 $fieldDefinition[] = "'notNull' => true";
+            } else {
+                if (!$field->isPrimary()) {
```

Comment:
Code looks good. Can you add a test to cover a primary field where isNotNull returns false.

Reply:
```diff
-            } else {
-                if (!$field->isPrimary()) {
+            } elseif (!$field->isPrimary()) {
```